ViewModels/UsersViewModels/TeachersViewModel.cs

```diff
@@ -0,0 +1,72 @@
+﻿using BD_oneLove.Models;
+using BD_oneLove.Tools;
+using BD_oneLove.Tools.Managers;
+using BD_oneLove.Views.UsersViews;
+using System;
+using System.Collections.Generic;
+using System.Linq;
```

Comment:
Unused usings?

Reply:
you are not applevovan to tell about them :)

Reply:
I am not, but you'd rather remove them

Reply:
![photo_2019-05-03_21-36-17](https://user-images.githubusercontent.com/37458100/57158238-98371680-6deb-11e9-8cfb-df3ba9a2f0e7.jpg)